docs/core/encoding.md

```diff
@@ -67,6 +67,23 @@ Note, there are length-prefixed variants of the above functionality and this is
 typically used for when the data needs to be streamed or grouped together
 (e.g. `ResponseDeliverTx.Data`)
 
+#### Authz authorizations
+
+Since the `MsgExec` message type can contain different messages instances, it is important that developers
+add the following code inside the `init` method of their module's `codec.go` file:
+
+```go
+import "github.com/cosmos/cosmos-sdk/codec/legacy"
+
+init() {
+    // Register all Amino interfaces and concrete types on the global Amino codec so that this can later be
+    // used to properly serialize x/authz MsgExec instances
+    RegisterLegacyAminoCodec(legacy.Cdc)	
+}
+```
+
+This will allow the `x/authz` module to properly serialize and de-serializes `MsgExec` instances using Amino.
```

Comment:
maybe let's add a note that this is only needed for ledger signing, which is an important use case.

Reply:
@robert-zaremba done